.github/workflows/ci.yml

```diff
@@ -84,8 +84,8 @@ jobs:
 
     - name: Run psalm
       if: success() || failure()
-      run: vendor/bin/psalm --output-format=github
+      run: psalm --output-format=github
```

Comment:
psalm and phpstan where not part of the devDependencies, so the build failed. i readded them here to get the build green. Open for other solutions as well.

Reply:
This is fine.